Sources/MapboxMaps/Location/InterpolatedLocation.swift

```diff
@@ -0,0 +1,33 @@
+import CoreLocation
+
+internal struct InterpolatedLocation: Equatable {
```

Comment:
You need a CLHeading to create a Location and CLHeading doesn't have any public constructors. Rather than trying to subclass it to add a constructor, it was simpler to create this alternative structure.